### ISSUE TITLE
Making the taint feel more magical.

### ISSUE THREAD

**Issue by Argis13** [Azanor/thaumcraft-suggestions#62]
To me the taint always seemed more biological infestation, like real world red tides (look it up) than a symptom of the very essence of reality being corrupted. It just seems too mundane in nature for, say a person that has never played thaumcraft to think "this isn't a problem that can't be solved by just blowing a bunch of this stuff up and breaking the rest." then imagine their shock when it just comes back. It is a corruption of reality, and I would like it to feel that way, not just a big fungus. I would like distortions which teleport you like an enderman, unbreakable blocks placed randomly in twisting columns, and degradation of armor and weapons, instead of just potion effects and purple tentacles. 
Feel free to drop your own ideas, because my own are _very_ vague.

**Comment by MalkContent** [comment]
I really like the teleportation idea. Maybe flash some strange patterns/pictures/have a short menacing vignette when it happens. Idea sprouting from half life 1 when you get teleported around the alien worlds at the start. Maybe have floating barely visible "instabilities" kind of like hot air. Get ported when you walk in that.
And as for armor degradation: it should happen through a potion effect though :P otherwise people probably wont even notice it.
Personally I feel like taint is fine. Big taint biomes look amazing. I'd probably just add sound effects like the croaking of the very earth as the taint warps it. Or some alien groaning/clicking sounds. Y'know no specific source, just creepy noises.
And I'm hoping tainted animals can eventually be implemented in a way that you can see more of them :)

**Comment by Argis13** [comment]
I didn't say they don't look good already, they do. I just want them to look good in a more magical way. But I get your point.

**Comment by Azanor** [comment]
Taint and flux in general is getting a rework.

**Comment by MalkContent** [comment]
sweet

**Comment by Argis13** [comment]
That's good, thanks for replying.

**Comment by angeltxilon** [comment]
I also think that effects of magic and bad magic should have more relation with physic and less with biology.

**Comment by carso150** [comment]
Yeah, I would love too see the taint being less euclid and more strange and unnatural
